cfe_ts_crc_version.h

```diff
@@ -40,14 +42,19 @@
 #define CFE_TS_CRC_MINOR_VERSION 1  /*!< @brief Minor version number */
 #define CFE_TS_CRC_REVISION      99 /*!< @brief Revision version number. Value of 99 indicates a development version.*/
```

Comment:
REVISION should be 0 instead of MISSION_REV being 00. Can use to set version string to development vs release.